tests/test_optimisation_model.py

```diff
@@ -123,16 +122,15 @@ def test_optimiser_compute():
     assert export_revenue == 0
 
 
+@pytest.mark.skipif(not pyo.SolverFactory("cbc").available(), reason="Skipping test because CBC is not available.")
 def test_optimisation_solve():
```

Comment:
Why does this end in an error?

test_optimisation_model.py::test_optimisation_solve 
Process finished with exit code 134 (interrupted by signal 6: SIGABRT)

Reply:
Method override issues.